content/en/security/cloud_workload_security/workload_security_rules.md

```diff
@@ -58,9 +58,7 @@ Detection rules run in the Datadog backend after events are sent over as logs. T
 
 ## Create custom rules
 
-In addition to the default rules, you can write custom Agent and detection rules. Custom Agent rules are deployed to the Agent in a policy separate from the default one.
-
-**Note**: At this time, Remote Configuration is only available for default rules. Custom rules must be manually deployed to the Datadog Agent.
+In addition to the default rules, you can write custom Agent and detection rules. Custom Agent rules are deployed to the Agent in a custom policy separate from the default one. The custom policy contains custom Agent rules as well as [default rules](#disable-default-agent-rules) that have been disabled.
```

Comment:
```suggestion
In addition to the default rules, you can write custom Agent and detection rules. Custom Agent rules are deployed to the Agent in a custom policy separate from the default one. The custom policy contains custom Agent rules as well as [default rules that have been disabled](#disable-default-agent-rules).
```
This is a nit, but I think it's better if the "disabled" part is inside the link or the user might think they're just being sent to a default rules section.